ImageLoader/JPEGImageOperations/JPEGImageOperations.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+import TurboJPEG
+ 
+#if os(iOS) || os(macOS) || os(tvOS) || os(watchOS)
+    import Darwin
+#elseif os(Android) || os(Linux)
+    import Glibc
+#elseif os(Windows)
+    import ucrt
+#else
+    #error("C library not known for the target OS")
+#endif
+
+public enum pixelFormat: Int32 {
+    case RGB888 = 0 // TJPF_RGB
+    case BGR888 = 1 // TJPF_BGR
+    case RGBA8888 = 2 // TJPF_RGBA
+    case BGRA8888 = 3 // TJPF_BGRA
+    case ARGB8888 = 4 // TJPF_ARGB
+    case ABGR8888 = 5 // TJPF_ABGR
+    case RGBA8880 = 6 // TJPF_RGBX
+    case BGRA8880 = 7 // TJPF_BGRX
+    case ARGB0888 = 8 // TJPF_XRGB
+    case ABGR0888 = 9 // TJPF_XBGR
+    case YUV400 = 10 // TJPF_GREY
+
+    var channelCount: Int32 {
+        switch self {
+        case .RGB888:
+            return 3
+        case .BGR888:
+            return 3
+        case .RGBA8888:
+            return 4
+        case .BGRA8888:
+            return 4
+        case .ARGB8888:
+            return 4
+        case .ABGR8888:
+            return 4
+        case .RGBA8880:
+            return 3
+        case .BGRA8880:
+            return 3
+        case .ARGB0888:
+            return 3
+        case .ABGR0888:
+            return 3
+        case .YUV400:
+            return 1
+        }
+    }
+}
+
+public class ImageData {
+    let height: Int32
+    let width: Int32
```

Comment:
```suggestion
    let height: CInt
    let width: CInt
```

I assume that these were `Int32` because they come from the C API?  If so, it is probably better to sugar this as `CInt`.  Otherwise, prefer `Int`.